core/frontend/src/components/kraken/InstalledExtensionCard.vue

```diff
@@ -199,6 +223,14 @@ export default Vue.extend({
       const total_kb = system_information.system?.memory?.ram?.total_kB
       return total_kb ?? undefined
     },
+    main_disk(): number | undefined {
+      const disks = system_information.system?.disk
+      return disks?.find((sensor) => sensor.mount_point === '/')
+    },
+    main_disk_size(): number {
+      const value = this.main_disk?.total_space_B ?? 0
+      return value / 1024 // Move to kB
+    },
```

Comment:
My brain is not working well today. but the typing here looks wrong.

if `main_disk()` is a number, how can we access `total_space_B` of it in the second method?

And why is CI not catching this?